### PR TITLE
nixos/glib-networking: init

### PR DESCRIPTION
**Pull request by worldofpeace**
Note that we were previously didn't have glib-networking
in systemd.packages so the PacRunner was non-functional.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/modules/services/desktops/gnome3/glib-networking.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    };
+
+  };
+
+  ###### implementation
+
+  config = mkIf config.services.gnome3.glib-networking.enable {
+
+    environment.systemPackages = [ pkgs.gnome3.glib-networking ];
```
Comment (jtojnar) [inline nixos/modules/services/desktops/gnome3/glib-networking.nix]: Why is this necessary?

Comment (worldofpeace) [reply nixos/modules/services/desktops/gnome3/glib-networking.nix]: Dropped.

**Comment by matthewbauer** [comment]
I'm not against doing this, but I wonder how useful glib-networking is as a standalone module? I mean most users just want *gnome* they aren't going to care about fine tuned things like glib networking service. Perhaps there is some useful part in making things more modular though. But until we have something like https://github.com/NixOS/rfcs/pull/22 there will be a cost for new modules (albeit very small).

**Comment by worldofpeace** [comment]
> I'm not against doing this, but I wonder how useful glib-networking is as a standalone module? I mean most users just want _gnome_ they aren't going to care about fine tuned things like glib networking service. Perhaps there is some useful part in making things more modular though. But until we have something like [NixOS/rfcs#22](https://github.com/NixOS/rfcs/pull/22) there will be a cost for new modules (albeit very small).

I have the same gripe that I don't feel it's useful as a standalone module.
I mainly just want to share this configuration across the multiple desktop modules that need it, and be able to change it in one place.
